### PR TITLE
ENT-2428: Updated image url field for cornerstone and degreed

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
JIRA: https://openedx.atlassian.net/browse/ENT-2428

**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
- [ ] Version pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)

### CODE REVIEW

File: integrated_channels/cornerstone/exporters/content_metadata.py [hunk 1]
```diff
@@ -98,6 +98,18 @@ def transform_estimated_hours(self, content_metadata_item):
 
         return duration
 
+    def transform_image(self, content_metadata_item):
```
Comment (georgebabey) [inline integrated_channels/cornerstone/exporters/content_metadata.py]: Now that we have this same method in 3 different files, I think we should either move this to a utils helper or the base class.

Comment (irfanuddinahmad) [reply integrated_channels/cornerstone/exporters/content_metadata.py]: Added to utils

File: integrated_channels/utils.py [hunk 2]
```diff
@@ -136,3 +136,16 @@ def strfdelta(tdelta, fmt='{D:02}d {H:02}h {M:02}m {S:02}s', input_type='timedel
...
     return f.format(fmt, **values)
+
+
+def get_image_url(content_metadata_item):
+    """
+    Return the image URI of the content item.
+    """
+    image_url = ''
+    if content_metadata_item['content_type'] == 'program':
+        image_url = content_metadata_item.get('card_image_url')
```
Comment (christopappas) [inline integrated_channels/utils.py]: probably not a big deal, but figured i would ask: is it possible (and do we want it to be) that this function can return None and not empty string if the `card_image_url` or `image_url` fields don't exist on the dict for some reason?

File: integrated_channels/degreed/exporters/content_metadata.py [hunk 3]
```diff
@@ -78,13 +78,7 @@ def transform_image(self, content_metadata_item):
         """
         Return the image URI of the content item.
         """
-        image_url = ''
```
Comment (ziafazal) [inline integrated_channels/degreed/exporters/content_metadata.py]: @irfanuddinahmad I'm wondering why we previously has different transformation logic for `degreed` and `SAP SF`

File: integrated_channels/utils.py (hunk 2 above)
Comment (irfanuddinahmad) [reply integrated_channels/utils.py]: @christopappas there is some logic around None ... 
https://github.com/edx/edx-enterprise/blob/a1974db6be53da3e4a11265de24d3a6f0b1e6824/integrated_channels/integrated_channel/exporters/content_metadata.py#L138

Comment (ziafazal) [reply integrated_channels/utils.py]: @irfanuddinahmad cornerstone does not accept values with `None` so we are skipping those items where we have a None value.

Comment (irfanuddinahmad) [reply integrated_channels/utils.py]: Thanks @ziafazal

**Pull request merged**